packages/relay/src/lib/relay.ts

```diff
@@ -124,7 +124,11 @@ export class RelayImpl implements Relay {
 
     client.setTransportSecurity(process.env.CLIENT_TRANSPORT_SECURITY === 'true' || false);
 
-    logger.info(`SDK client successfully configured to ${JSON.stringify(hederaNetwork)} for account ${client.operatorAccountId}`);
+    const sdkRequestTimeout = parseInt(process.env.SDK_REQUEST_TIMEOUT || '');
+    client.setRequestTimeout(Number.isInteger(sdkRequestTimeout) ? sdkRequestTimeout : 1000);
```

Comment:
Is 1000 (I assume this is milliseconds) a good default value?  Do we know how much time calls to the consensus nodes typically take?

Reply:
Yes it's milliseconds and good question.  Locally it seems to work with no issues.  Tests pass.  I am currently running/testing against hashio services and will have results back soon.

Reply:
Check w services and sdk team

Reply:
I asked around and it seems that 10000 would be a good default value to start with.  I updated the README, test .env files and helm charts accordingly.